bundles/org.openhab.core.io.rest.core/src/main/java/org/openhab/core/io/rest/core/internal/item/ItemResource.java

```diff
@@ -831,7 +859,7 @@ private JsonObject buildStatusObject(String itemName, String status, @Nullable S
     /**
      * helper: Response to be sent to client if a Thing cannot be found
      *
-     * @param thingUID
+     * @param itemname
```

Comment:
Nice catch. Can you also adjust `a Thing` two lines above to `an item`?

Reply:
Will do, being the friendly boy scout now who cleans up the camp ;-) I went over all missing descriptions and added them.

there are more hints, though and one in particular that could be optimized:

In the following case String.trim() is actually ignored (this is what my IDE tells me). 
```
List<String> acceptedMediaTypes = Arrays.stream(Objects.requireNonNullElse(mediaType, "").split(","))
                .peek(String::trim).collect(Collectors.toList());
```             

and it also says that "collect(Collectors.toList())" can be simply changed to .toList()

Should I change that?

Reply:
The IDE is fully correct. `.peek` should be a `.map`, please don't ask my why I did such a silly thing :-). Using `.toList` is always preferred in Java 17.

Reply:
It is actually interesting. I changed it to 

List<String> acceptedMediaTypes = Arrays.stream(Objects.requireNonNullElse(mediaType, "").split(","))
                .map(String::trim).collect(Collectors.toList());

and the IDE says

'collect(toList())' can be replaced with 'toList()'

So, I even let the IDE do so:

<img width="336" alt="image" src="https://user-images.githubusercontent.com/5937600/210366576-53202838-c787-4e96-942a-02b3fedc72e0.png">

which results into 

<img width="974" alt="image" src="https://user-images.githubusercontent.com/5937600/210366685-c8a8d07d-0884-4bf8-9109-c4d3fd5e893f.png">


Cannot resolve method 'toList' in 'Stream'

Reply:
Does it compile? Sometimes my IDE also does strange things...

Reply:
Yes, it does, but to be honest, I don't want to commit something that is shown red in my IDE ;-)

Reply:
Which IDE is it?From the colors it looks a bit like IntelliJ. Did you set the project JDK to 17?